services/tests/e2e/authorization/auth-jwt.spec.ts

```diff
@@ -8,14 +7,18 @@ import {
   UpdateSchemasMutationResponse,
 } from '../../helpers/registry-request-builder';
 import GraphQLErrorSerializer from '../../utils/graphql-error-serializer';
+import { getToken } from '../../helpers/get-token';
 import { schema, policies, getUserQuery, arbitraryDataQuery } from './auth-jwt.schema';
 
-const gatewayClient = createGatewayClient();
-const registryClient = createRegistryClient();
+const gatewayClient = new GraphQLClient('http://localhost:8080/graphql');
+const registryClient = new GraphQLClient('http://localhost:8090/graphql');
 
 describe('authorization', () => {
-  beforeAll(() => {
+  beforeAll(async () => {
     expect.addSnapshotSerializer(GraphQLErrorSerializer);
+
+    const accessToken = await getToken();
+    gatewayClient.setHeader('Authorization', `Bearer ${accessToken}`);
```

Comment:
We should probably have this in `beforeEach` since some tests override this with a specific token, and we want to restore the standard one for the next tests.
We can generate the token only once in `beforeAll` and then just set it every time in `beforeEach`